plugins/hashsum_download/server/__init__.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+###############################################################################
+
+from girder.api import access
+from girder.api.describe import describeRoute, Description
+from girder.api.rest import RestException
+from girder.api.v1.file import File
+from girder.constants import AccessType
+
+
+class HashedFile(File):
+
+    SupportedAlgorithms = [
```

Comment:
@jcfr @girder/developers Indeed, I've noticed that the Flake8 style tests do not enforce camelCase for variable names. We should file this as a bug.